xdg_test.go

```diff
@@ -212,7 +213,18 @@ func TestNonExistentRuntimeDir(t *testing.T) {
 	xdg.Reload()
 	require.Equal(t, nonExistentRuntimeDir, xdg.RuntimeDir)
 
-	p, err := xdg.RuntimeFile("app.pid")
-	require.NoError(t, err)
-	require.Equal(t, filepath.Clean(os.TempDir()), filepath.Dir(p))
+	for _, runtimeFile := range runtimeFiles {
+		suggestedPath, err := xdg.RuntimeFile(runtimeFile)
+		require.NoError(t, err)
+		require.Equal(t, true, strings.HasPrefix(suggestedPath, os.TempDir()))
+
+		f, err := os.Create(suggestedPath)
+		require.NoError(t, err)
+		require.NoError(t, f.Close())
+		defer os.Remove(suggestedPath)
```

Comment:
This line should be after creating the suggestedPath, so after the require.NoError

Reply:
That's true, the `defer` statement should go before the file closing. Thank you, I somehow missed that.

Reply:
Would you like to open a PR with the suggested changes?

Reply:
No need, go ahead.

I'm on phone now